<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientEditView</class>
 <widget class="QWidget" name="PatientEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑患者信息</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/dp4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>776</width>
     <height>356</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ID:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="dbEditID">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 1px 15px 1px 3px;
  border:1px  solid;
}
</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>姓名：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dbEditName">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 1px 15px 1px 3px;
  border:1px  solid;
}</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>身份证：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="dbEditIDCard">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 1px 15px 1px 3px;
  border:1px  solid;
}</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>性别：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="dbComboSex">
            <property name="styleSheet">
             <string notr="true">QComboBox{
  color:#666666;
  font-size:14px;
  padding: 1px 15px 1px 3px;
  border:1px  solid;
} 

QComboBox QAbstractItemView::item{
	height:36px;
	color:#666666;
	padding-left:9px;
	background-color:#FFFFFF;
}
QComboBox QAbstractItemView::item:hover{ //悬浮
  background-color:#409CE1;
  color:#ffffff;
}
QComboBox QAbstractItemView::item:selected{//选中
  background-color:#409CE1;
  color:#ffffff;
}
QComboBox:on { 
      padding-top: 3px;
      padding-left: 4px;
  }
  QComboBox::down-arrow:on { 
      top: 1px;
      left: 1px;
  }</string>
            </property>
            <item>
             <property name="text">
              <string>男</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>女</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>年龄：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="dbSpinAge">
            <property name="styleSheet">
             <string notr="true">QSpinBox
 {
padding: 1px 15px 1px 3px;
  border:1px  solid;
}

QSpinBox::up-button //QSpinBox的向上按钮
{
subcontrol-origin: border;//subcontrol-origin指定原始矩形内子控件的对齐方式
subcontrol-position: top right; /* position at the top right corner */
width: 16px; 
border-width: 1px;
}

QSpinBox::up-arrow //向上箭头
{
width: 7px;
height: 7px;
}

QSpinBox::down-button //向下按钮
{
subcontrol-origin: border;
subcontrol-position: bottom right; /* position at bottom right corner */
width: 16px;
border-width: 1px;//边界宽度
border-top-width: 0;//边界顶部宽度
}

QSpinBox::down-arrow //向下的箭头
{
width: 7px;
height: 7px;
}

</string>
            </property>
            <property name="maximum">
             <number>199</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>出生日期：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="dbDateEditDOB">
            <property name="styleSheet">
             <string notr="true">QDateEdit
 {
padding: 1px 15px 1px 3px;
  border:1px  solid;
}

QDateEdit::up-button //QSpinBox的向上按钮
{
subcontrol-origin: border;//subcontrol-origin指定原始矩形内子控件的对齐方式
subcontrol-position: top right; /* position at the top right corner */
width: 16px; 
border-width: 1px;
}
QDateEdit::up-arrow //向上箭头
{
width: 7px;
height: 7px;
}

QDateEdit::down-button //向下按钮
{
subcontrol-origin: border;
subcontrol-position: bottom right; /* position at bottom right corner */
width: 16px;
border-width: 1px;//边界宽度
border-top-width: 0;//边界顶部宽度
}

QDateEdit::down-arrow //向下的箭头
{
width: 7px;
height: 7px;
}

</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>身高：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="dbSpinHeight">
            <property name="styleSheet">
             <string notr="true">QSpinBox
 {
padding: 1px 15px 1px 3px;
  border:1px  solid;
}

QSpinBox::up-button //QSpinBox的向上按钮
{
subcontrol-origin: border;//subcontrol-origin指定原始矩形内子控件的对齐方式
subcontrol-position: top right; /* position at the top right corner */
width: 16px; 
border-width: 1px;
}

QSpinBox::up-arrow //向上箭头
{
width: 7px;
height: 7px;
}

QSpinBox::down-button //向下按钮
{
subcontrol-origin: border;
subcontrol-position: bottom right; /* position at bottom right corner */
width: 16px;
border-width: 1px;//边界宽度
border-top-width: 0;//边界顶部宽度
}

QSpinBox::down-arrow //向下的箭头
{
width: 7px;
height: 7px;
}

</string>
            </property>
            <property name="suffix">
             <string>cm</string>
            </property>
            <property name="maximum">
             <number>299</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>体重：</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="dbSpinWeight">
            <property name="styleSheet">
             <string notr="true">QSpinBox
 {
padding: 1px 15px 1px 3px;
  border:1px  solid;
}

QSpinBox::up-button //QSpinBox的向上按钮
{
subcontrol-origin: border;//subcontrol-origin指定原始矩形内子控件的对齐方式
subcontrol-position: top right; /* position at the top right corner */
width: 16px; 
border-width: 1px;
}

QSpinBox::up-arrow //向上箭头
{
width: 7px;
height: 7px;
}

QSpinBox::down-button //向下按钮
{
subcontrol-origin: border;
subcontrol-position: bottom right; /* position at bottom right corner */
width: 16px;
border-width: 1px;//边界宽度
border-top-width: 0;//边界顶部宽度
}

QSpinBox::down-arrow //向下的箭头
{
width: 7px;
height: 7px;
}

</string>
            </property>
            <property name="suffix">
             <string>kg</string>
            </property>
            <property name="maximum">
             <number>699</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>手机号：</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="dbEditMobile">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 1px 15px 1px 3px;
  border:1px  solid;
}</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>创建时间：</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="dbCreatedTimeStamp">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 1px 15px 1px 3px;
  border:1px  solid;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btSave">
            <property name="text">
             <string>保存</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/img/dbsave.png</normaloff>:/new/prefix1/img/dbsave.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btCancel">
            <property name="text">
             <string>取消</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/img/cancel.png</normaloff>:/new/prefix1/img/cancel.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>130</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
